index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <title>Renoma</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="hamburger__wrapper"><img class="hamburger" src="images/Hamburger.png" alt="Hamburger butten image"></div>
+            <div class="vertical-divider"></div>
+            <div class="count-label__wrapper"><p class="count-label">2</p></div>
+            <div class="cart__wrapper"><img class="cart" src="images/cart.png" alt="Cart image"></div>
+            <div class="logo__wrapper"><p class="logo">Renome</p></div>
```

Comment:
Also how we discuss with you need to rename this class `class="logo-wrapper"` and the same for others.